### PR TITLE
Expand documentation

### PR DESCRIPTION
**Pull request by dmitryduev**
In this PR: 

- Detailed quick start docs from the user perspective
- GH issue templates
- Updated `readme.md`
- Description of catalogs available on `Kowalski`
- Technical implementation details
- Fixed broken integration tests

Here are some screen shots/captures from the quick start section:

invite

invited

landing

groups-1

groups-request-admission

candidates-small

source-small

alerts

finder

### CODE REVIEW

File: doc/catalogs.md [hunk 1]
```diff
@@ -0,0 +1,1219 @@
+# Kowalski's collection schemas
+
+Fritz hosts a number of astronomical catalogs on its Kowalski backend.
+Below you will find a list of available catalogs and the approximate contents of
+individual entries therein (example documents per MongoDB collection).
+Note that the is no schema enforcement, so specific fields may or may not exist
```
Comment (guynir42) [inline doc/catalogs.md:6]:
```suggestion
Note that there is no schema enforcement, so specific fields may or may not exist
```

File: doc/catalogs.md [hunk 2]
```diff
@@ -0,0 +1,1219 @@
...
+for individual documents.
+
+
```
Comment (guynir42) [inline doc/catalogs.md:9]: Maybe this is obvious for more experienced users, but I'm not sure what are these JSON examples. I guess I'm not sure what "example documents per MongoDB collection" actually means.

These catalogs include all the cross matches on Kowalski, or just a sample?

File: doc/run_your_own.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+If you are using the default settings, go to `http://localhost:5000/` --
+you should see a few real alerts that passed test filters among the displayed sources.
+
+## Launching Fritz
+
+```
+./fritz run
+```
+
+## Shitting down Fritz
```
Comment (guynir42) [inline doc/run_your_own.md:49]:
```suggestion
## Shutting down Fritz
```

**Review by guynir42 — APPROVED** [review]

**Review by acrellin — APPROVED** [review]

File: doc/catalogs.md (hunk 1 above)
Comment (dmitryduev) [reply doc/catalogs.md:6]: Fixed

File: doc/catalogs.md (hunk 2 above)
Comment (dmitryduev) [reply doc/catalogs.md:9]: This is everything that is available on Kowalski. It's actually not yet exposed to everyone, only to the users who have K accounts.

File: doc/run_your_own.md (hunk 3 above)
Comment (dmitryduev) [reply doc/run_your_own.md:49]: ROFL

Comment (dmitryduev) [reply doc/run_your_own.md:49]: thanks for catching this :)

**Review by dannygoldstein — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Platform information:**
+ - Fritz version:  (find this in the [About](https://fritz.science/about) page)
+ - Browser
+   - [ ] I'm browsing on a desktop
+   - [ ] I'm browsing on my phone
+
```
Comment (profjsb) [inline .github/ISSUE_TEMPLATE/bug_report.md:33]: - could also have tablet as an option (with phone and computer these are the three main options)
- bug could be API related so maybe want to allow this to.

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+title: "Short feature description"
+labels: enhancement
+assignees: ''
+
+---
+
+Please fill out relevant sections below; remove those that are unused.
+
+**Feature Summary**
+Please give a clear, concise description of the proposed feature.  We are working on a [Minimal Viable Product](https://en.wikipedia.org/wiki/Minimum_viable_product), so please motivate why this feature is considered a high priority.
```
Comment (profjsb) [inline .github/ISSUE_TEMPLATE/feature_request.md:13]: Are we past MVP now? Might want to consider different language.

File: doc/run_your_own.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
+# Run your own Fritz
+
+Clone the repo from `GitHub`:
+```bash
+git clone --recursive https://github.com/fritz-marshal/fritz.git && cd fritz
+```
+
+All operations are started via the `fritz` script.
+Make sure the requirements to run it are met, e.g.:
+
```
Comment (profjsb) [inline doc/run_your_own.md:10]: Maybe ask the user to start their own conda env here so they don't cross-pollute the installed python modules

**Review by profjsb — CHANGES_REQUESTED** [review]
Few small changes here. For a future PR you might consider autogenerating/autopopulating the version strings of the oft-changing ML models, CLU db, etc.

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 5 above)
Comment (dmitryduev) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Fixed

File: doc/run_your_own.md (hunk 6 above)
Comment (dmitryduev) [reply doc/run_your_own.md:18]: Added some text on that.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 4 above)
Comment (dmitryduev) [reply .github/ISSUE_TEMPLATE/bug_report.md:35]: Fixed.

**Comment by dmitryduev** [comment]
Although this is unrelated to the main topic of this PR, I also fixed the integration tests! They broke when https://github.com/cesium-ml/baselayer/pull/178 went in, which caused the following: SP would not properly initialize when spun up with docker-compose with the default cmd. I overcame this by running `make run` instead of `make run_production` when doing `./fritz run --init --dev`, which ensures the correct behavior + added checks on whether SP is up and running before trying integration testing.
@stefanv pinging you here since we should address this in SP, as now the default fresh docker-based deployment fails.

**Pull request merged**